src/java/voldemort/store/quota/QuotaUtils.java

```diff
@@ -32,4 +46,47 @@ public static String makeQuotaKey(String storeName, QuotaType quotaType) {
         }
         return quotaTypes;
     }
+
+  private static FileBackedCachingStorageEngine getQuotaStore(StoreRepository repoistory) {
+    FileBackedCachingStorageEngine quotaStore =
+        (FileBackedCachingStorageEngine) repoistory.getStorageEngine(QUOTA_STORE);
+
+    if (quotaStore == null) {
+      throw new VoldemortApplicationException("Could not find the quota store for Store " + QUOTA_STORE);
+    }
+
+    return quotaStore;
+  }
+
+  private static ByteArray convertToByteArray(String value) {
+    try {
+      return new ByteArray(value.getBytes("UTF8"));
+    } catch (UnsupportedEncodingException ex) {
+      throw new VoldemortApplicationException("Error converting key to byte array " + value, ex);
+    }
+  }
+
+  public static Long getQuota(String storeName, QuotaType type, StoreRepository repoistory) {
```

Comment:
That's great. It made no sense to use the AdminClient locally on ourselves... (y)